### PR TITLE
bump version of component-boilerplate for batch refresh update

### PR DESCRIPTION
**Pull request by 00salmon**
## Description of the change

> Implements component-boilerplate update from: https://github.com/saasquatch/program-tools/pull/232

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

 - Jira issue number: (PUT IT HERE)
 - Process.st launch checklist: (PUT IT HERE)

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist
- [ ] "Ready for review" label attached to the PR and reviewers pinged on Slack

### Code review 

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

File: packages/bedrock-components/src/components/sqb-widget/useWidget.ts [hunk 1]
```diff
@@ -50,14 +50,14 @@ const WIDGET_LOAD_EVENT = gql`
...
-
+  const locale = useLocale();
   const [fetch, { data }] = useLazyQuery<GetWidget>(GET_WIDGET);
   const [sendLoadEvent] = useMutation(WIDGET_LOAD_EVENT);
-
   const analyticsEventSent = useRef(false);
 
   if (props.trackLoads && !analyticsEventSent.current && userIdent !== undefined) {
     analyticsEventSent.current = true;
+    console.log('SENDING LOAD EVENT');
```
Comment (00salmon) [inline packages/bedrock-components/src/components/sqb-widget/useWidget.ts:60]:
```suggestion
```

File: packages/bedrock-components/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/bedrock-components",
-  "version": "1.3.0",
+  "version": "1.3.1-2",
```
Comment (00salmon) [inline packages/bedrock-components/package.json:3]:
```suggestion
  "version": "1.3.1",
```

File: packages/bedrock-components/package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/bedrock-components",
-  "version": "1.3.0",
+  "version": "1.3.1-2",
```
Comment (00salmon) [inline packages/bedrock-components/package-lock.json:3]:
```suggestion
  "version": "1.3.1",
```

**Comment by DerekSiemens** [comment]
This all looks good to me once versions are updated for the non pre-release deploy and boilerplate is deployed

File: packages/bedrock-components/src/components/sqb-widget/useWidget.ts [hunk 4]
```diff
@@ -75,17 +76,17 @@ export function useWidget(props: SqbWidget) {
     // No auth required
     !props.requireAuth ||
     // Or auth required and logged in
-    userIdent !== undefined;
+    (userIdent !== undefined && locale !== undefined);
+
   useEffect(() => {
     if (props.widgetType && canLoad) {
+      console.log(locale);
```
Comment (loganvolkers) [inline packages/bedrock-components/src/components/sqb-widget/useWidget.ts:83]:
```suggestion
```

File: packages/bedrock-components/src/components/sqb-widget/sqb-widget.feature [hunk 5]
```diff
@@ -23,6 +23,23 @@ Feature: Widget
             | p/referral-a/w/referrerWidget |
             | global-widget                 |
 
+    @motivating
+    Scenario Outline: Widgets translations are loading according to a users locale
```
Comment (DerekSiemens) [inline packages/bedrock-components/src/components/sqb-widget/sqb-widget.feature:27]: Missing no locale case

File: packages/bedrock-components/CHANGELOG.md [hunk 6]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.3.1] - 2022-02-23
+
+### Changed
+
+- Changed components:
+  - \<sqb-widget>
+    - Now supports translations, returning the localized widget
```
Comment (DerekSiemens) [inline packages/bedrock-components/CHANGELOG.md:16]: Missing change to program section!

File: packages/bedrock-components/src/index.html [hunk 7]
```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
     <title>Bedrock Components</title>
     <script src="https://fast.ssqt.io/npm/@samsquatch/vanilla-components@latest/dist/widget-components.js"></script>
-    <script src="https://fast.ssqt.io/npm/@saasquatch/mint-components/dist/mint-components/mint-components.js"></script>
+    <script src="https://fast.ssqt.io/npm/@saasquatch/mint-components@next/dist/mint-components/mint-components.js"></script>
```
Comment (DerekSiemens) [inline packages/bedrock-components/src/index.html:8]: Making a note that we will want to swap this back before launch

Comment (00salmon) [reply packages/bedrock-components/src/index.html:8]: ah meant to correct that, good catch

File: packages/bedrock-components/CHANGELOG.md (hunk 6 above)
Comment (00salmon) [reply packages/bedrock-components/CHANGELOG.md:16]: added!

**Review by DerekSiemens — APPROVED** [review]

**Pull request merged**